### ISSUE TITLE
Inverted Mask Convention in VACE vs. WAN2.1 and Potential Implications

### ISSUE THREAD

**Issue by pekinghk** [Wan-Video/Wan2.1#479]
First, thank you for your excellent work on VACE! It’s an impressive contribution to video frame interpolation.  

During experiments with VACE for **first/last frame interpolation**, I noticed an interesting discrepancy:  
1. **Mask Convention Inversion**:  
   - The base model WAN2.1-14B uses `0` for **frames to generate** and `1` for **frames to preserve**.  
 <img width="1510" height="312" alt="Image" src="https://github.com/user-attachments/assets/0749edbc-30f5-43c2-a5ed-86975a5da862" />
   - However, VACE’s implementation **reverses this convention** (`1` = generate, `0` = preserve).  
<img width="735" height="248" alt="Image" src="https://github.com/user-attachments/assets/fbbf2a51-e666-4761-b5c3-bae13bcb5ae5" />

2. **Potential Adaptation Concerns**:  
   - VACE uses **Context Adapter Tuning**, which (as per the paper) **does not finetune the Video Tokens module** inherited from WAN2.1.  
   - Given the inverted mask handling and lack of Video Tokens training:  
     - Could this mismatch introduce **latent artifacts** (e.g., positional confusion or feature misalignment)?  
     - Are there ablation studies validating that WAN2.1’s pretrained Video Tokens robustly handles inverted masking without degradation?  

### Request  
Could you clarify:  
- The **design rationale** behind flipping the mask convention?  
- Whether this inversion was **empirically validated** against the original convention?  
- Any insights on how **static Video Tokens** interact with inverted masks during inference?  

Thanks again for your work—I look forward to your insights!
